File: develop/lab-3-dynamodb/dynamoDBPythonLab/reservations_data_uploader_test.py
# Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights
# reserved.

import unittest
import reservations_data_uploader
import warnings


class ReservationsDataUploaderTest(unittest.TestCase):

    def test_reservations_uploader(self):
        warnings.simplefilter("ignore", ResourceWarning)
        numFailures = reservations_data_uploader.load_reservations_data()
        self.assertEqual(0, numFailures)

if __name__ == '__main__':
    unittest.main()

File: develop/lab-3-dynamodb/dynamoDBJavaLab/src/test/java/com/amazonaws/lab/ReservationsDataUploaderTest.java
package com.amazonaws.lab;
// Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights reserved.

import static org.junit.Assert.*;

import org.junit.Test;

public class ReservationsDataUploaderTest {

	@Test
	public void test() throws Exception {
		ReservationsDataUploader.main(new String[0]);
		assertEquals(1000, ReservationsDataUploader.numItemsAdded, 0);
	}
}
